uxarray/__init__.py

```diff
@@ -1,2 +1,17 @@
 from .grid import *
 from .helpers import *
+
+
+def __version__():
```

Comment:
Even though this is a very small function, please include a simple docstring describing the functionality. It can be simply a single sentence like the example below:

https://github.com/UXARRAY/uxarray/blob/efca32ed3bd8edb0a81edfba18561938b06200db/uxarray/grid.py#L703-L705